.github/workflows/ci.yml

```
@@ -0,0 +1,34 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  merge_group:
+  workflow_dispatch:
+
+jobs:
+  check:
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v4
+      - name: Setup tools
+        uses: jdx/mise-action@v2
+      - name: Install dependencies
+        run: bun install --frozen-lockfile
+      - name: Lint and format with Biome
```

NITS: import sortingが行われていることも明示したほうが良いかもしれません。suggestionは公式ドキュメントの表現を参考にしました。

https://biomejs.dev/ja/reference/cli/#biome-ci

```suggestion
      - name: Run formatter, linter and import sorting with Biome
```